### DISCUSSION TITLE
✅ COMPLETED - RfC - improve the UX of fundraising info in the django CMS admin interface

### DISCUSSION THREAD

**Discussion by macolo** [django-cms/django-cms#8018] [General]
@fsbraun  and @NicolaiRidani  brainstormed ways to create new touchpoints with django CMS users to raise awareness for our dCA sponsorships / memberships. We realized that we already have two existing opportunities within django CMS to share information with users / devs:

- Whenever there’s an update, a message appears in the django admin interface
- Whenever a new project is created, a successful installation page is shown (with some information) at the beginning which is immediately blocked by the "start new page wizard".

Therefore our proposal is:

- Add a default line about our sponsorship program to the update message (has to be implemented into the core)
- Make use of the successful installation page and include information about our sponsorship program...make it feel like a proper welcome page
- Disable the immediate pop-up of the new page wizard and instead highlight the “Start New Page” button on the successful installation page which will then trigger the wizard

**Comment by macolo** [comment]
Based on work from @NicolaiRidani and @fsbraun I'd like to contribute the following drafts:

Short version
```
A) quick and easy: Support the django CMS Fellowship through [Github Sponsors](https://github.com/sponsors/django-cms). 
B) Here are [other ways to contribute](https://www.django-cms.org/en/give-back-to-the-community/).
```

The following is (with minor adaptions) what we have on https://github.com/sponsors/django-cms - does it make sense to use this same content or do you think this is too long?

```
**👋 By the way - django CMS needs your help!** 

Please contribute! Your funding directly benefits the product, mainly through the [django CMS Fellowship Program](https://www.django-cms.org/en/fellowship-program/)! Here is how:

A) If you look for a quick way for yourself or your organisation to donate, you can do so on [Github Sponsors](https://github.com/sponsors/django-cms). 

B) Alternatively, you and/or your organisation can [join the django CMS Association as a member](https://www.django-cms.org/en/memberships/), especially if you would like to:

- be involved in annual votings on product strategy and budget allocation
- receive the annual financial reports of the association
- get in touch with other members.
- get elected into the association's Board

C) See here for [other ways to contribute](https://www.django-cms.org/en/give-back-to-the-community/).

**Who is behind django CMS?**
django CMS is managed by the django CMS Association, which is a non-profit organization that was founded in 2020 and is based in Zurich, Switzerland. We are a team of volunteers and we are the Guardians of django CMS. Find out more [about the association here](https://www.django-cms.org/en/about-us/).
```

**Comment by fsbraun** [comment]
#8057 implements a welcome page with the three points from above. I suggest we start improving from there. :-) 
![image](https://github.com/user-attachments/assets/497ca3c0-83a4-497f-b1b4-6a576254a96a)

**Reply by macolo** [reply]
that looks awesome @fsbraun ! 🙌
